src/nft/components/profile/view/ProfilePage.tsx

```diff
@@ -202,10 +202,11 @@ export const ProfilePage = () => {
           borderRadius="12"
           paddingX="16"
           paddingY="12"
+          background="backgroundModule"
```

Comment:
This will be off, Styled Components use `background` (not `backgroundModule`) for setting a background in dark/light mode. Unless this is rendered in some other container, there will be a difference in shade. 

Ideally, the easiest would be to rewrite this component to Styled Components and reference proper theme variable from there! 

Check my PR for some direction on how this can be done: #5309

Reply:
ok!

Just wanted to comment that we're shipping a change to master that changes background color. On your branch, background and backgroundModule are the same (gray800). But soon, background will be gray850.

Reply:
ok cool - good to know